releasetasks/templates/firefox/final_verify.yml.tmpl

```diff
@@ -52,7 +52,7 @@
 
         extra:
             {{ common_extras(taskname=buildername, locales=["null"], platform="null") | indent(12)}}
-
+            {{ task_notifications(taskname="{} {} final verification".format(branch, channel), completed=["releasetasks"], failed=["releasetasks"], exception=["releasetasks"]) | indent(12) }}
             {# TODO: decide what to use here...
```

Comment:
Something tells me that you need to pull latest update. I believe, I removed these TODOs.